packages/mb-lib/src/text/text.lib.ts

```
@@ -17,3 +17,32 @@ export function toSlug(str: string): string {
+  
+  // If string is already in correct format, return it
+  if (slugRegex.test(str)) {
+    return str;
+  }
+  
+  return str
+    // Convert to lowercase
+    .toLowerCase()
+    // Replace accented characters with regular ones
+    .normalize('NFD')
+    .replace(/[\u0300-\u036f]/g, '')
```

Biome

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)